test/unit/server/services/procedures/chart/chart.spec.js

```diff
@@ -32,11 +32,65 @@ describe(utils.suiteName(__filename), function() {
             'grid',
             'isTimeSeries',
             'timeInputFormat',
-            'timeDisplayFormat'
+            'timeDisplayFormat',
+            'logscale',
         ];
         assert.deepEqual(opts.map(i => i[0]), expectedOpts);
     });
 
+    describe('logscale', function() {
+        it('should allow empty axes', function(){
+            const [, options] = Chart._parseDrawInputs([], {logscale: ''});
+            assert.equal(options.logscale, undefined);
+        });
+        it('should allow empty axes', function(){
+            const [, options] = Chart._parseDrawInputs([], {logscale: ['']});
+            assert.equal(options.logscale.axes, '');
+            assert.equal(options.logscale.base, 10);
+        });
+        it('should parse string logscale (using base 10)', function(){
+            const [, options] = Chart._parseDrawInputs([], {logscale: 'x'});
+            assert.equal(options.logscale.axes, 'x');
+            assert.equal(options.logscale.base, 10);
+        });
+        it('should parse string logscale (using base 10)', function(){
```

Comment:
duplicate test names

Reply:
Oh, ok, I was treating them more like `expect` messages in rust tests. In that case, I'll merge the related ones together.